src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,61 @@
+.clock {
+  @extend %position;
```

Comment:
This placeholder contains margin. Don't use margins on BEM blocks

Reply:
it is for positioning elements, same like default values... in next task i will not use this:)

Reply:
Let's do it in this task